module-2/backend/week_one.md

```diff
@@ -5,30 +5,63 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+ * *GET - this is a request that a file/page be returned from the server ('gimmie what choo got')*
+ * *POST - this is a way to pass information from the client to the server ('here, have a form')*
+ * *PUT - this is a way to request that a resource get updated ('here, my name is Bob, not Robert')*
+ * *DELETE - this is a way to pass a destroy request, oftentimes intending it to go to a database ('Destroy Robert!')*
+ * *PATCH - this is another way to update a record - like PUT, gets used with a form/edit page ("I'm not 70, I was born in the 70's")*
 2. What is Sinatra?
+ * *Sinatra is a gem.  It's a web server for ruby and freaking amazing.*
 4. What is MVC?
+ * *Model View Controller.  It's a way of organizing web applications in an object oriented manner so that the responsibilities are cleanly separated.*
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+ * *I would say it's because a model (example) has been found to work so we're following that, and it maintains consistency across projects.  This way, if you open my Sinatra-based application, you can instanly navigate the file structure and understand the structure of the application.*
 6. What types of variables are accessible in our view templates without explicitly passing them?
+ * *instance variables (a-la @cats)*
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
-    erb :index
+    erb :index, locals: { count: 1, name: 'Mr. Ed' }

```

Comment:
here you're passing the local variable `count`, not the instance variable specified in the question - but i think you have a good understanding of this given your answer to question six 👍